app/Controllers/StaticPagesController.js

```diff
@@ -0,0 +1,47 @@
+const catchAsync = require('../utils/catchAsync');
+const path = require('path');
+const ApiError = require('../utils/ApiError.js');
+const httpStatus = require('http-status');
+
+const renderHandler = catchAsync(async (res, page) => {
+  const options = { root: path.join(__dirname, '../../') };
+
+  // check if page should be rendered from a static file or should be redirected to another one
+  if (page.type === 'file') {
+    res.sendFile(page.location, options, (err) => {
+      if (err) {
+        throw new ApiError(httpStatus.NOT_FOUND, 'Page was not found');
+      }
+    });
+  } else {
+    res.redirect(page.location);
+  }
+});
+
+const renderPage = catchAsync(async (req, res, pageValues) => {
```

Comment:
I meant on discord (see at the bottom for the second part):
```suggestion
const getRenderPageHandler = (pageValues) => catchAsync(async (req, res) => {
```